ui/src/app/views/workspace/component.html

```diff
@@ -1,7 +1,11 @@
 <div *ngIf="!notFound else notFoundMessage">
   <div *ngIf="!workspaceLoading; else workspacePageLoading" style="display: flex;">
     <h2>{{workspace.name}}</h2>
-    <clr-icon class="editing-icon is-solid" shape="pencil" (click)="edit()"></clr-icon>
+    <span *ngIf="accessLevel === workspaceAccessLevel.OWNER then writePermissions"></span>
+    <span *ngIf="accessLevel === workspaceAccessLevel.WRITER then writePermissions"></span>
+    <ng-template #writePermissions>
+        <clr-icon class="editing-icon is-solid" shape="pencil" (click)="edit()"></clr-icon>
```

Comment:
this could be

```
<clr-icon *ngif="accessLevel === workspaceAccessLevel.Owner || acce..." /* other attrs */></clr-icon>
```

or even better:
```
/* in template */
<clr-icon *ngIf="hasPermission"></clr-icon>

/* in component */
get hasPermission() {
  return this.accessLevel === WorkspaceAccessLevel.OWNER 
    || this.accessLevel === WorkspaceAccessLevel.WRITER;
}
```

and then you don't need to attach a copy of `WorkspaceAccessLevel` to the component (as in line 106).